lib/swagger/info.rb

```diff
@@ -0,0 +1,35 @@
+class Swagger::Info
+  attr_accessor :fields
+
+  def initialize(fields)
+    @fields = Hashie::Mash.new(fields)
+  end
+
+  delegate :description,
+           :title,
+           :version,
+           to: :fields
+
+  delegate :email,
+           :name,
+           :url,
+           to: :contact,
+           prefix: true
+
+  delegate :name,
+           :url,
+           to: :license,
+           prefix: true
+
+  def terms_of_service
+    fields.termsOfService
+  end
+
+  def contact
+    fields['contact'] || { }
```

Comment:
nice! I feel like we could do more stuff like this to avoid this conditionals in the views